server/src/models/user.js

```diff
@@ -31,6 +31,9 @@ const user = new mongoose.Schema({
     profile: {
         type: String,
         required: false
+    },
+    interest: {
```

Comment:
name it "interests" instead, since it's a list

Reply:
Just curious, what effect does that have or is it just style?

Reply:
just style